modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc

```diff
@@ -123,6 +123,10 @@ endif::[]
 
 * Work with the user to identify files that can be deleted from the `/opt/app-root/src` directory on their notebook server to free up their existing storage space.
 
+[NOTE]
+--
+When you delete files using the JupyterLab file explorer, the files move to the hidden `/opt/app-root/src/.local/share/Trash/files` folder in the PVC for the notebook. To free up storage space for notebooks, you must permanently delete these files.
```

Comment:
@grainnejenningsRH I wonder if we mean the persistent volume (PV) for the notebook, rather than the persistent volume claim (PVC), which is the underlying Kubernetes resource used to request that storage. Can you double-check that with Jorge?

Either way, introduce the acronym in parentheses by spelling it out in full, as I did above.

Reply:
I'm not sure about the correct term, but when creating  a workbench, the UI just says "persistent storage".  Could we just say _the files move to the hidden `/opt/app-root/src/.local/share/Trash/files` folder in the persistent storage for the notebook_?

Reply:
> I'm not sure about the correct term, but when creating a workbench, the UI just says "persistent storage". Could we just say _the files move to the hidden `/opt/app-root/src/.local/share/Trash/files` folder in the persistent storage for the notebook_?

That sounds good to me, @jgarciao.